### DISCUSSION TITLE
MQTT separate topic

### DISCUSSION THREAD

**Discussion by gerdluthe** [johanmeijer/grott#109] [Q&A]
Hello,

i would like to get the mqtt values as an separate topic like

mqtt.0.energy.growatt.pvstatus
mqtt.0.energy.growatt.pvpower
......
is this possible to get this instead of an JSON table?

Regards
gerd

**Comment by johanmeijer** [comment]
Hi Gerd,

The challenge would be to sent all separate topics with a certain performance (if there are a lot of values).

What we maybe can do is create an extension that only send specific values.

I will have a look at that. Give me some time there are a lot of requests at this moment. You are of welcome to create your own extension (see https://github.com/johanmeijer/grott/wiki/Extensions) of course.

**Comment by ZonPV** [comment]
I don't know with what purpose... Can you explain why you would do this? In what system would you read these values?

I should be possible with a function in Node-Red to split the values.  Don't know the status of your project, but I can send you an example

**Reply by gerdluthe** [reply]
thanks for your reply. i'd managed the problem in the meanwhile with javascript to split the json-array.
